### PR TITLE
Clustered REST Health Check Endpoint Docs

### PR DESCRIPTION
**Pull request by emre-aydin**

### CODE REVIEW

File: docs/modules/ROOT/pages/clustered-rest.adoc [hunk 1]
```diff
@@ -73,13 +133,27 @@ curl http://localhost:8080/rest/clusters
...
+[#cluster-resource]
 == Cluster Resource
 
-This resource returns information related to the provided cluster name;
-address of the oldest cluster member, the expiration
-time in milliseconds (since epoch) of the license key assigned for the cluster
-and the cluster type (`IMDG` or `JET`). It returns `-1` for license expiration time
-if no license is assigned.
+This resource returns information related to the provided cluster name:
+
```
Comment (Serdaro) [inline docs/modules/ROOT/pages/clustered-rest.adoc:140]: Maybe it is a good idea to put the names of the returned property names before the explanations below?.. I leave it to you.

**Review by Serdaro — COMMENTED** [review]
Minor comments

File: docs/modules/ROOT/pages/clustered-rest.adoc (hunk 1 above)
Comment (emre-aydin) [reply docs/modules/ROOT/pages/clustered-rest.adoc:140]: I was in between for that one. Since you mention it as well, I guess that's a good idea.

**Comment by emre-aydin** [comment]
@Serdaro thanks for the review, addressed all.

**Review by Serdaro — APPROVED** [review]

**Pull request merged**